templates/order/order-details.php

```diff
@@ -109,6 +109,5 @@
  */
 do_action( 'woocommerce_after_order_details', $order );
 
-if ( $show_customer_details ) {
```

Comment:
maybe we can improve the `$show_customer_details` instead of removing it, to display it when this is a guest order?